rakam-postgresql/src/main/java/org/rakam/postgresql/analysis/PostgresqlMetastore.java

```diff
@@ -362,6 +363,11 @@ public void createProject(String project)
         }
     }
 
+    @Override
+    public List<String> getAttributes(String project, String collection, SchemaField field, Optional<LocalDate> startDate, Optional<LocalDate> endDate, Optional<String> query) {
+        return null;
```

Comment:
After we finished the implementation of Presto, we should also implement this method in this PR.

Reply:
Implemented.